### ISSUE TITLE
Tests fail in 32bit mode

### ISSUE THREAD

**Issue by csoeger** [gap-packages/NormalizInterface#36]
I did test 32bit compilation on a 64bit system for issue #34 and it does work for me but some tests fail. The problem here is that some integer values are then returned as a fraction with an additional `/1`.
A shortened snippet from the log:
```
# Expected output:
Multiplicity = 72
# But found:
Multiplicity = 72/1
```

**Comment by csoeger** [comment]
This difference is gone when I remove the ifdef on line 82 in scr/normaliz.cc and do the case `size == 1` also for 32 bit.
https://github.com/gap-packages/NormalizInterface/blame/master/src/normaliz.cc#L82

@fingolfin Is this a sensible solution, or is the distinction really needed?

**Comment by csoeger** [comment]
I just added suggested conversion tests (not yet in test format) and the actually fail on 64bit but run through on 32 bit!

The test consist of a matrix containing positive and negative values around powers of 2 and the extreme rays that are computed should be the same. Rows which differ are printed.

**Comment by fingolfin** [comment]
I converted the test to .tst format, and fixed the bug, see 18c6b2f410e429be2de0ef519ba63e3fe7426e33
